src/EnergyPlus/Pumps.cc

```diff
@@ -892,10 +887,9 @@ void GetPumpInput(EnergyPlusData &state)
 
         // Input the optional schedule for the pump
         if (thisInput->cAlphaArgs(6).empty()) { // Initialized to zero, don't get a schedule for an empty
-            thisPump.flowRateSched = Sched::GetScheduleAlwaysOn(state);
+            thisPump.flowRateSched = nullptr;
```

Comment:
It's already null

        Sched::Schedule *flowRateSched = nullptr;                          // Flow rate modifier schedule, blank/missing --> AlwaysOn